app/src/main/java/com/mborowiec/antennapod/activity/MainActivity.java

```diff
@@ -542,40 +544,54 @@ public Snackbar showSnackbarAbovePlayer(int text, int duration) {
     }
 
     /**
-     * Handles the incoming deep link and loads the relevant fragment.
-     * By default, opens the "Add Podcast" feature.
+     * Handles the deep link incoming via App Actions and according to the query
```

Comment:
Nitpicking: What about splitting the long sentence, e.g., like

Handles the deep link incoming via App Actions. Performs an in-app search or opens the relevant feature of the app depending on the query.